src/pytorch_lightning/callbacks/checkpoint.py

```diff
@@ -0,0 +1,9 @@
+from pytorch_lightning.callbacks.callback import Callback
+
+
+class Checkpoint(Callback):
+    r"""
+    This is the base class for Model checkpointing. Expert users may want to subclass it in case of writing
+    custom :class:`~pytorch_lightning.callbacks.model_checkpoint.ModelCheckpoint` callback, so that
```

Comment:
it wouldn't be a custom ModelCheckpoint though, because it doesn't subclass it.

Reply:
You're right, done